practical-penguins/trivia_tavern/trivia_runner/templatetags/settings_tag.py

```diff
@@ -0,0 +1,12 @@
+# borrowed from https://stackoverflow.com/questions/433162
```

Comment:
brilliant. well done.

Reply:
Very nice to see credit given in any location where code was borrowed/used. :-)